src/Uno.UI.RemoteControl.Host/Uno.UI.RemoteControl.Host.csproj

```diff
@@ -20,7 +20,7 @@
 		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton" />
 		<PackageReference Include="Microsoft.IO.RecyclableMemoryStream" Version="2.3.2" />
 		<PackageReference Include="StreamJsonRpc" Version="2.14.24" />
```

Comment:
There are newer versions of Microsoft.IO.RecyclableMemoryStream and StreamJsonRpc, should we be updating them too. Unlike Uno itself where we keep version updates to a minimum, I would have thought for dev tooling we should be pushing to latest where possible?